MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -887,6 +900,12 @@ - (void)observeValueForKeyPath:(NSString*) path
         if ([self isPlayerInErrorState]) {
             [self dispatchError];
         }
+    } else if (context == MUXSDKAVPlayerItemPlaybackBufferEmptyObservationContext) {
+        if ([_playerItem isPlaybackBufferEmpty] && _player.timeControlStatus != AVPlayerTimeControlStatusPlaying && [self isPausedWhileAirPlaying]) {
+            // We erroneously detected a pause when in fact we are rebuffering. This *only* happens in AirPlay mode
+            [self dispatchPlay];
+            [self dispatchPlaying];
```

Comment:
If we detect this scenario, kick ourselves back into the playing state so that we actually call `startBuffering` in the time observer: https://github.com/muxinc/mux-stats-sdk-avplayer/blob/0dd6a8582503fd05f0a9f15a5900d25955da6abf/MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m#L73